test/base/TextTest.ooc

```diff
@@ -92,49 +92,50 @@ TextTest: class extends Fixture {
 			expect(parts[3] == "split")
 		})
 		this add("converting to integer", func {
-			expect(Text new("1") toInt() == 1)
-			expect(Text new("-1") toInt() == -1)
-			expect(Text new("-932") toInt() == -932)
-			expect(Text new("871") toInt() == 871)
-			expect(Text new("bad") toInt() == 0)
-			expect(Text new("bad") toInt~inBase(16) == 11 * 16 * 16 + 10 * 16 + 13)
-			expect(Text new("123one") toInt() == 123)
-			expect(Text new("101") toInt() == 101)
-			expect(Text new("101") toInt~inBase(2) == 5)
-			expect(Text new("101") toInt~inBase(8) == 8 * 8 + 1)
-			expect(Text new("101") toInt~inBase(7) == 7 * 7 + 1)
-			expect(Text new("654") toInt~inBase(6) == 0)
-			expect(Text new("654") toInt~inBase(7) == 4 + 5 * 7 + 6 * 7 * 7)
-			expect(Text new("BEEF") toInt() == 0)
-			expect(Text new("BEEF") toInt~inBase(16) == 48879)
-			expect(Text new("BEEF") toInt~inBase(16) == Text new("beef") toInt~inBase(16))
-			expect(Text new("0xff") toInt() == 255)
-			expect(Text new("0x11") toInt() == 17)
-			expect(Text new("0xAA") toInt() == 170)
-			expect(Text new("0xffZZZ") toInt() == 255)
-			expect(Text new("0xDEADBEEF") toULong() == 3735928559)
-			expect(Text new(INT_MAX toString()) toLong() == INT_MAX)
-			expect(Text new("-9") toULong() == 0)
-			expect(Text new("-9") toLLong() == -9)
-			expect(Text new("-9") toLong() == -9)
+			expect(Text new("1") toInt(), is equal to(1))
+			expect(Text new("-1") toInt(), is equal to(-1))
+			expect(Text new("-932") toInt(), is equal to(-932))
+			expect(Text new("871") toInt(), is equal to(871))
+			expect(Text new("bad") toInt(), is equal to(0))
+			expect(Text new("bad") toInt~inBase(16), is equal to(11 * 16 * 16 + 10 * 16 + 13))
+			expect(Text new("123one") toInt(), is equal to(123))
+			expect(Text new("101") toInt(), is equal to(101))
+			expect(Text new("101") toInt~inBase(2), is equal to(5))
+			expect(Text new("101") toInt~inBase(8), is equal to(8 * 8 + 1))
+			expect(Text new("101") toInt~inBase(7), is equal to(7 * 7 + 1))
+			expect(Text new("654") toInt~inBase(6), is equal to(0))
+			expect(Text new("654") toInt~inBase(7), is equal to(4 + 5 * 7 + 6 * 7 * 7))
+			expect(Text new("BEEF") toInt(), is equal to(0))
+			expect(Text new("BEEF") toInt~inBase(16), is equal to(48879))
+			expect(Text new("BEEF") toInt~inBase(16), is equal to(Text new("beef") toInt~inBase(16)))
+			expect(Text new("0xff") toInt(), is equal to(255))
+			expect(Text new("0x11") toInt(), is equal to(17))
+			expect(Text new("0xAA") toInt(), is equal to(170))
+			expect(Text new("0xffZZZ") toInt(), is equal to(255))
+			expect(Text new("0xDEADBEEF") toULong(), is equal to(3735928559))
+			expect(Text new(INT_MAX toString()) toLong(), is equal to(INT_MAX))
+			expect(Text new("-9") toULong(), is equal to(0))
+			expect(Text new("-9") toLLong(), is equal to(-9))
+			expect(Text new("-9") toLong(), is equal to(-9))
```

Comment:
If any of these expects fail, it's going to be hard to figure out which one. Split this up into smaller test functions with a smaller number of expects. I think you can figure out how to split it up logically.